modules/nodes-nodes-aws-autoscaler-LC-and-ASG.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-aws-autoscaler.adoc
+
+[id='nodes-nodes-aws-autoscaler-primed-images-{context}']
```

Comment:
@mburke5678 - it should be _ {context}. Applies to all ids in your modules. In assemblies, it looks ok. I know that there is an ongoing discussion about it. For the moment, let's be consistent and use _{context}.